_playwright-tests/helpers/loginHelpers.ts

```diff
@@ -67,6 +67,13 @@ export const storeStorageStateAndToken = async (page: Page, fileName: string) =>
   await page.waitForTimeout(100);
 };
 
+export const logInWithRHELOperatorUser = async (page: Page) =>
+  await logInWithUsernameAndPassword(
+    page,
+    process.env.RHEL_OPERATOR_USERNAME,
+    process.env.RHEL_OPERATOR_PASSWORD,
```

Comment:
these environment variables for credentials, should also be added to the `MandatoryEnvVariables` list and also to the steps that create the .env files in the GH workflows (both the PR checks one and also the nightly) 🥸

Reply:
@swadeley these environment variables are still missing from the github workflow steps 😅 [[1](https://github.com/content-services/content-sources-frontend/blob/8a2522ff5b788bf7de07c7d9a0f2b7a0e12992ad/.github/workflows/playwright-actions.yml#L41-L47)] [[2](https://github.com/content-services/content-sources-frontend/blob/8a2522ff5b788bf7de07c7d9a0f2b7a0e12992ad/.github/workflows/nightly-stage-test-action.yml#L53-L65)]